tests/integration/README.md

```diff
@@ -38,10 +38,14 @@ deleted manually.
 Copy `.env.sample` to `.env`, put the respective provider configurations in the
 environment variables and source it, `source .env`.
 
```

Comment:
The different variables for each provider in `.env.sample` are commented out.
We can remind users here to uncomment it (or change the `.env.sample` so that they aren't commented).

Reply:
> put the respective provider configurations in the environment variables and source it

should mean the same as it's talking about sourcing env vars.